docs/about/overview/index.mdx

```diff
@@ -15,3 +15,14 @@ When a receiving radio captures a packet, it checks to see if it has heard that
 For each message a radio rebroadcasts, it marks the "hop limit" down by one. When a radio receives a packet with a hop limit of zero, it will not rebroadcast the message.
 
 The radio will store a small amount of packets (around 30) in its memory for when it's not connected to a client app. If it's full, it will replace the oldest packets with newly incoming text messages only.
+
+## What is a mesh?
+
+At the radio level a Meshtastic mesh is a set of nodes that share the same LoRa spreading factor, center frequency, and bandwidth. A node can only be in one radio mesh; it will not see or respond to messages from nodes using different values for these settings. For a mesh to form, nodes need to share the same values. 
+
+These values are grouped into "presets" that can be easily chosen in the LoRa configuration section. Presets make it easy for nodes to configure the same radio parameters.
+
+Sitting on top of this radio mesh are Channels. A logical mesh is formed by a Channel with a particular name and encryption key. The default channel in a radio mesh is Channel 0 with a blank "name" and an ecryption key of AQ==. 
+
+Nodes can belong to a maximum of 8 Channels in the radio mesh. A custom Channel can be created for use by a specific group. Only nodes configured with the same Channel name and encryption key will be able to read and display messages on that Channel. However, all nodes in the radio mesh will receive and retransmit messages (depending on their Role) regardless of the Channel settings for the message.
```

Comment:
As @pdxlocations also pointed out, it depends on the Rebroadcast Mode whether they'll route packets for other Channels. Not sure we want that detail here, but otherwise we could add "by default".